src/go/configinfo/service_info.go

```diff
@@ -607,18 +609,31 @@ func (s *ServiceInfo) extractApiKeyLocations(method *methodInfo, parameters []*c
 	method.ApiKeyLocations = append(method.ApiKeyLocations, headerNames...)
 }
 
-func (s *ServiceInfo) processTypes() {
-	// Create snake name to JSON name mapping.
+func (s *ServiceInfo) ProcessTypes() error {
+	// Create snake name to JSON name mapping (and validate against duplicates).
```

Comment:
why capitalize? It should called inside service_info.go only

Reply:
Right, thanks. Got confused by test structure